tidy3d/components/grid/grid_spec.py

```diff
@@ -18,6 +18,61 @@
 from ...exceptions import SetupError
 from ...constants import MICROMETER, C_0, fp_eps
 
+# Default Courant number reduction rate in Benkler's scheme
+DEFAULT_COURANT_REDUCTION_BENKLER = 0.3
+
+
+class ConformalMeshSpec(Tidy3dBaseModel, ABC):
+    """Base class defining conformal mesh specifications."""
+
+    @cached_property
+    def courant_ratio(self) -> float:
+        """The scaling ratio applied to Courant number so that the courant number
+        in the simulation is ``sim.courant * courant_ratio``.
+        """
+        return 1.0
+
+
+class StaircasingConformalMeshSpec(ConformalMeshSpec):
+    """Simple staircasing scheme based on
+    [Taflove, The Electrical Engineering Handbook 3.629-670 (2005): 15.].
+    """
+
+
+class HeuristicConformalMeshSpec(ConformalMeshSpec):
+    """Slight improvement from the staircasing scheme where the field component near PEC
```

Comment:
Are you claiming it's an improvement based on sphere results?

One thing that bothers me about the heuristic mesh spec is that the PEC boundary is not effectively continuous. Generally a very unpredictable handling...

But, it's good to have some official description about it as this is the legacy handling. :) I think this description is OK.

Reply:
Now after running more tests, I would remove "slight improvement". In sphere case, it improves, but not in many other cases.